NIPTool/models/converters.py

```diff
@@ -0,0 +1,36 @@
+def empty_str(x):
```

Comment:
```suggestion
from typing import Optional

def empty_str(x: str) -> Optional[str]:
```
I suggest that this function takes a string as argument and skip the check if x is a string below, makes the functionality more clear. Now it does not only "Covert empty string to None" but also "Checks if argument is a string". Check if string outside this function

Reply:
When using type hints we should use Optional when a function return something or None ref: https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#when-you-re-puzzled-or-when-things-are-complicated

Reply:
Cool. Didn't know about Optional! After a long conversation with @patrikgrenfeldt we concluded that the converter is unnesesary. Ill make a function empty_str_to_none() and Ill use optional!